sklearn/calibration.py

```diff
@@ -1061,8 +1062,7 @@ def plot(self, *, ax=None, name=None, ref_line=True, **kwargs):
         if ax is None:
             fig, ax = plt.subplots()
 
-        name = self.name if name is None else name
-        self.name = name
```

Comment:
Agree that this is weird, maybe it's left over from a previous iteration/API and I forgot to change.